PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+
+# What you've done
+...
+
+# Why have you done it
+...
+
+# Screenshot (if visual)
+...
+
+# Any testing carried out
+...
```

Comment:
@gayanhewa Sorry I still didn't get this :) The sample PR has an automatic resolver word right? That's what I'm suggesting to include. AFAIK that should be a part of the template.